R/util.R

```diff
@@ -6,3 +6,59 @@
 is_directory <- function(x) {
   file.info(x, extra_cols = FALSE)$isdir
 }
+
+
+is_call <- function(x, name) {
+  is.recursive(x) && is.name(x[[1]]) && as.character(x[[1]]) == name
```

Comment:
```
is.call(list(1, 2)) # FALSE
is.recursive(list(1, 2)) # TRUE
```

I tend to use is.recursive here as the more general one but also because in this context it reads better (something we might recurse into). Note that `is_call` here as I'm writing it here is really ("is specific call")

Reply:
Thanks! Actually looking at docs again `is.recursive returns TRUE if x has a recursive (list-like)` is a good hint